.pre-commit-hooks.yaml

```diff
@@ -1,16 +1,49 @@
--   id: apigentools-validate
-    name: Validate OpenAPI specification files
-    language: python
-    entry: apigentools
-    args:
-      - validate
-    description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository.
-    pass_filenames: false
--   id: container-apigentools-validate
-    name: Validate OpenAPI specification files using container wrapper
-    language: python
-    entry: container-apigentools
-    args:
-      - validate
-    description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository using container wrapper.
-    pass_filenames: false
+- id: validate
+  name: Validate OpenAPI specification files
+  language: python
+  entry: apigentools validate
+  description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository.
+  pass_filenames: false
+- id: templates
```

Comment:
Do we explicitly need templates if its inside generate?

Reply:
No, but @bkabrda wants separate steps in CI.